source/ed/connectors/fusio_parser.cpp

```diff
@@ -227,6 +228,15 @@ void RouteFusioHandler::handle_line(Data& data, const csv_row& row, bool) {
     if (is_valid(geometry_id_c, row))
         ed_route->shape = find_or_default(row.at(geometry_id_c), data.shapes);
 
+    if (is_valid(destination_id_c, row)){
+        const auto search = gtfs_data.stop_area_map.find(row.at(destination_id_c));
+        if (search != gtfs_data.stop_area_map.end()){
+            ed_route->destination =  search->second;
+        }else{
+            LOG4CPLUS_WARN(logger, "impossible to find destination: " + row[route_id_c]);
```

Comment:
Maybe you can add `row.at(destination_id_c)` in the message. And its better to use `<<` in logs:
```c++
LOG4CPLUS_WARN(logger, "impossible to find destination " << row.at(destination_id_c) << "for route " << row.at(route_id_c));